README.md

```diff
@@ -1,26 +1,119 @@
 # Ripple
 
-This is work-in-progress beta version.
+[![CircleCI](https://circleci.com/gh/dpc-sdp/ripple/tree/master.svg?style=svg&circle-token=242dc8445ab25fb88fe506609fd7065cd1f78f7c)](https://circleci.com/gh/dpc-sdp/ripple/tree/master)
+[![Vue.js 2.x](https://img.shields.io/badge/vue.js-2.x-green.svg?style=flat-square)](https://vuejs.org)
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lernajs.io/)
 
-You should check our [alpha](https://github.com/dpc-sdp/ripple/tree/v1.0.0-alpha.84)
-version at this stage.
 
-## Requirements
 
-- Install [nodejs](https://nodejs.org/en/)
-- Install [yarn](https://yarnpkg.com/en/docs/install)
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/dpc-sdp/ripple">
+    <img src="/docs/logo.png" alt="Logo" height="150">
+  </a>
 
-## Install
+  <h3 align="center">Ripple</h3>
+
+  <p align="center">
+    <br />
+    <a href="https://dpc-sdp.github.io/sdp-docs/ripple/"><strong>Documentation Site »</strong></a>
+    <br />
+    <br />
+    <a href="https://ripple.sdp.vic.gov.au/">Storybook</a>
+    ·
+    <a href="http://app-ripple-develop.lagoon.vicsdp.amazee.io">Reference Ripple site</a>
+  </p>
+</p>
+
+
+
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+
+* [About the Project](#about-the-project)
+  * [Built With](#built-with)
+* [Getting Started](#getting-started)
+  * [Requirements](#requirements)
+  * [Installation](#install)
+  * [Example app](#running-example-application)
+  * [Storybook](#running-storybook)
+* [Usage](#usage)
+* [Contributing](#contributing)
+* [License](#license)
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+Ripple is the presentational layer for building websites on the DPC Single Digital Presence platform. 
+It includes :
+
+- A reusable component library based upon an atomic design system.
+- Modules for the universal rendering framework [_Nuxt_](https://nuxtjs.org) to create websites using a [_Tide_](https://github.com/dpc-sdp/tide) Drupal backend.
+- Helper utilities to run UI tests on Ripple sites.
+
+
+### Built With
+
+* [Vue](https://vuejs.org/)
+* [Nuxt](https://nuxtjs.org)
+* [Storybook](https://storybook.js.org/)
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+Ripple includes both a component explorer using Storybook and an example reference application. 
+
+
+### Requirements
+
+- Install [nodejs](https://nodejs.org/en/) (^v10.0.0)
+- Install [yarn](https://yarnpkg.com/en/docs/install) (currently npm is unsupported)
+
+### Install
 
 Run `yarn install`
 
-## Running Example
+### Running Example Application
```

Comment:
I think this part is belong to 'CONTRIBUTING', as this is for local env setup of core devs work.

Users of Ripple should just want to know how to install Ripple into their project.

Reply:
They will not interact with "Ripple" they will interact with "@dpc-sdp/ripple-nuxt-tide" Which is where that documentation should be. But I have updated the Ripple README.md with basic usage for each package. I think this should be an overview of everything with links for more detail as required.